tests/src/main.rs

```diff
@@ -123,8 +123,8 @@ impl Runner {
                 "--crate-type dylib",
                 "-Zunstable-options",
                 "-Zcrate-attr=no_std",
-                "-Zcrate-attr=feature(register_attr,asm_const,asm_experimental_arch)",
-                "-Zcrate-attr=register_attr(spirv)",
+                "-Zcrate-attr=feature(register_tool,asm_const,asm_experimental_arch)",
+                "-Zcrate-attr=register_tool(rust_gpu)",
```

Comment:
As I was saying elsewhere, we need to just copy this into `spirv-builder` so that users don't need to use themselves (tho we should do only the `register_tool` feature, not `asm_const` - not sure about `asm_experimental_arch`, but if it's needed to use `asm!` *at all*, we might want to inject it anyway)

Reply:
It *is* in SpirvBuilder (no-std and register_tool), but the tests don't use SpirvBuilder.